iModelCore/ECPresentation/Tests/NonPublished/Integration/Content/ContentIntegrationTests.cpp

```diff
@@ -953,6 +953,194 @@ TEST_F(RulesDrivenECPresentationManagerContentTests, DescriptorOverride_FiltersB
     RulesEngineTestHelpers::ValidateContentSet(bvector<IECInstanceCP>{b1.get()}, *content);
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsitest
++---------------+---------------+---------------+---------------+---------------+------*/
+DEFINE_SCHEMA(Guid_AppliesExtendedDataRule, R"*(
+    <ECEntityClass typeName="A">
+        <ECProperty propertyName="GuidProp" typeName="binary" extendedTypeName="BeGuid" />
+    </ECEntityClass>
+)*");
+TEST_F(RulesDrivenECPresentationManagerContentTests, Guid_AppliesExtendedDataRule)

```

Comment:
I think I'd just add to `AppliesContentItemExtendedDataFromPresentationRules` test for testing extended data.

Reply:
fixed